azure-pipelines/testapps/Android-Complete AzureSample LocalDebug.yml

```diff
@@ -1,30 +1,31 @@
-# 'Allow scripts to access the OAuth token' was selected in pipeline.  Add the following YAML to any steps requiring access:
-#       env:
-#           MY_ACCESS_TOKEN: $(System.AccessToken)
-# Variable 'android:serverUrl' was defined in the Variables tab
+# File: azure-pipelines\testapps\Android-Complete AzureSample LocalDebug.yml
+# Description: Assemble Local Debug Azure Sample App APK
 # Variable 'build_type' was defined in the Variables tab
 # Variable 'distMsalVersion' was defined in the Variables tab
-# Variable 'ENV_VSTS_MVN_ANDROID_MSAL_USERNAME' was defined in the Variables tab
 # Variable 'msal_flavor' was defined in the Variables tab
+# Variable 'ENV_VSTS_MVN_ANDROID_MSAL_USERNAME' was defined in the Variables tab
 # Variable 'mvnAccessToken' was defined in the Variables tab
-# Variable 'test_repo_branch' was defined in the Variables tab
-# Variable 'test_repo_dir' was defined in the Variables tab
-# Cron Schedules have been converted using UTC Time Zone and may need to be updated for your location
-trigger: none
-schedules:
-- cron: 0 1 * * 1,2,3,4,5
-  branches:
-    include:
-    - shahzaibj/droidSetupDevexOnly
-  always: true
+# https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate
 name: $(date:yyyyMMdd)$(rev:.r)
+
+trigger: none
+pr: none
+
 resources:
   repositories:
   - repository: android-complete
     type: github
     name: AzureAD/android-complete
     ref: shahzaibj/droidSetupDevexOnly
     endpoint: ANDROID_GITHUB
+
+schedules:
+- cron: 0 1 * * 1,2,3,4,5
+  branches:
+    include:
+    - shahzaibj/droidSetupDevexOnly
```

Comment:
>shahzaibj/droidSetupDevexOnly [](http://example.com/codeflow?start=6&length=29)

should this be the default branch

Reply:
It wouldn't work on default branch yet until I merge my change (I hadn't merged because I also wanted some other changes to go in that PR but never got the chance...I guess I should open a PR with what I have and merge it)

Reply:
This is now merged...and we can change branch to `master` now.

Reply:
Done